tests/src/dir.common.props

```diff
@@ -30,13 +30,14 @@
 <!-- Setup the default output and intermediate paths -->
   <PropertyGroup>
     <BaseOutputPathWithConfig>$(ProjectDir)\..\binaries\tests\$(BuildOS).$(Platform).$(Configuration)\</BaseOutputPathWithConfig>
-    <BaseOutputPathWithConfig Condition="'$(__TestWorkingDir)' != ''">$(__TestWorkingDir)\</BaseOutputPathWithConfig>
+    <BaseOutputPathWithConfig Condition="'$(__TestBinDir)' != ''">$(__TestBinDir)\</BaseOutputPathWithConfig>
     <BinDir>$(BaseOutputPathWithConfig)\..</BinDir>
-    <BaseIntermediateOutputPath>$(BinDir)\intermediates\</BaseIntermediateOutputPath>
-    <__CMakeTestSlnDir Condition="'$(__CMakeTestSlnDir)' == ''">$(BaseOutputPathWithConfig)\CMake\</__CMakeTestSlnDir>
+    <BaseIntermediateOutputPath>$(ProjectDir)\..\binaries\tests\intermediates\</BaseIntermediateOutputPath>
+    <BaseIntermediateOutputPath Condition="'$(__ManagedTestIntermediatesDir)' != ''">$(__ManagedTestIntermediatesDir)\</BaseIntermediateOutputPath>
+    <__NativeTestIntermediatesDir Condition="'$(__NativeTestIntermediatesDir)' == ''">$(ProjectDir)\..\binaries\tests\intermediates\$(BuildOS).$(Platform).$(Configuration)\</__NativeTestIntermediatesDir>
```

Comment:
$(BaseOutputPathWithConfig) is used to get the location of tests everywhere. Can you construct the path from it.
So that overriding testdirectory locations are easier and not hardwired to a particular location ?

$(BaseOutputPathWithConfig)\..\intermediates\$(BuildOS).$(Platform).$(Configuration)